ldms/src/core/ldms_core.h

```diff
@@ -193,15 +195,31 @@ struct ldms_set_hdr {
 	uint8_t pad3;	/* data pad */
 	uint32_t card;		/* Size of dictionary */
 	uint32_t meta_sz;	/* size of meta data in bytes */
-	uint32_t data_sz;	/* size of metric values in bytes */
+	uint32_t data_heap_sz;	/* size of metric values in bytes */
 	uint32_t uid;           /* UID */
 	uint32_t gid;           /* GID */
 	uint32_t perm;          /* permission */
 	uint32_t array_card;    /* number of sets in the set array */
-	uint32_t reserved[8];	/* area reserved for compatible core updates */
```

Comment:
Did you determine that you didn't need to consume any of the reserved space? If so, please remove this field.

Reply:
It is already removed. The `red` part is the removed stuff.